src/main/treeDataProviders/dependenciesTree/dependenciesDataProvider.ts

```diff
@@ -50,8 +46,14 @@ export class DependenciesTreeDataProvider implements vscode.TreeDataProvider<Dep
         }
         try {
             this._scanInProgress = true;
-            this._treesManager.logManager.logMessage('Starting ' + (quickScan ? 'quick' : 'slow') + ' Xray scan...', 'INFO');
-            await this.repopulateTree(quickScan);
+            const credentialsSet: boolean = this._treesManager.connectionManager.areCredentialsSet();
+            this._treesManager.logManager.logMessage('Starting ' + (quickScan ? 'quick' : 'slow'), 'INFO');
+            if (!credentialsSet) {
+                this._treesManager.logManager.logMessage(' Xray scan...', 'INFO');
```

Comment:
```suggestion
                this._treesManager.logManager.logMessage('Xray scan...', 'INFO');
```

Reply:
Maybe it will be better to remove both 'Xray scan...' and 'GoCenter scan...' and log 'Starting scan...'. This is since we don't know if we are really in GoCenter mode.

Reply:
Yes, removed.